mainmatter/introduction/graphics_apis/index.tex

```diff
@@ -19,7 +19,7 @@ \subsection{Modern APIs}
 Usually the driver must pass some tests and requirements, for the \gls{API} to be officially supported on the device.   
 
 The most popular low-level graphics \glspl{API} during the last game console generation, lasting from 2005 to about 2017, have been Microsoft’s Direct3D, part of the DirectX multimedia API collection, and Khronos’ OpenGL. 
-Direct3D is designed by Microsoft to be used exclusively on the Microsoft Windows platform. 
+%Direct3D is designed by Microsoft to be used exclusively on the Microsoft Windows platform. 
 The underlying driver is partially implemented by Microsoft in a closed common runtime, which communicates with the hardware vendors actual \gls{GPU} driver through the Direct3D \gls{DDI}\cite{dxDDI}.
 On the other hand, OpenGL is designed by several hardware and software vendors in coorperation, and requires that manufacturers write their own library from scratch. 
 
```

Comment:
Bent's kommentar: "benifit or curse" er ikke håndteret

Reply:
Ellers hans: "det lyder som ret spekulation - har i noget evidens for dette udsagn?" kommentar (linje 45)